<h5>Bloco <em>finally</em></h5>
<p>
	Com o <em>finally</em>, que será colocado após o bloco do <em>Catch</em>, será sempre executado, mesmo que ocorra um <em>RuntimeException</em>. Isso pode ser útil, para caso tenha alguma função que queira que funcione de qualquer forma. Mesmo se não ocorrer qualquer problema, o <em>finally</em> ainda compilara o seu código.
</p>
<p>
	É possível que seja “jogado” (<em>throw</em>) mais do que uma exceção na declaração do <em>throw</em>, separando eles por vírgulas. Para pegar a outra exceção, basta adicionar um <em>Catch</em> extra, sem necessitar de um novo <em>Try</em>.
</p>